internal/ingress/store/fake_store.go

```diff
@@ -124,7 +125,18 @@ func NewFakeStore(
 			return nil, err
 		}
 	}
-
+	var ingressClassMatching annotations.ClassMatching
```

Comment:
This block is an exact copy of a matching block in `store.New()`.

A fake should not copy the logic of a real SUT it replaces, otherwise we risk testing the fake instead of testing the SUT.